.gitignore

```diff
@@ -3,3 +3,8 @@
 .import/
 .godot/
 .vim/
+
+# Ignore backup files of various editors
+*~
+#*#
+.#*
```

Comment:
What do these rules do exactly?

Reply:
Line 9 Emacs auto-save files
Line 10 Emacs locks (mechanism to detect simultaneous edits by multiple Emacs users).
 
I think I should've escaped "#"  though. Just a sec.

Reply:
Ah, that makes more sense. I've been wondering about the comments in there.